src/elements/header/script.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+var document, window;
+(function(){
```

Comment:
Why are we using this syntax? Webpack will wrap the code for us. This seems backward.